src/common/styles/icons.m.css

```diff
@@ -0,0 +1,148 @@
+@font-face {
+	font-family: 'dojo2';
+	src:
+		url('fonts/dojo2.ttf?k76fx7') format('truetype'),
+		url('fonts/dojo2.woff?k76fx7') format('woff'),
```

Comment:
Again, this is the limited output available from icomoon. I think that if we have a premium account then we might get access to the `woff2` but I'm not sure. Might be worth finding out if they have an open source offering outside of their paid model. @dylans any idea here?